src/overlays/actors/ovl_Obj_Hsblock/z_obj_hsblock.c

```diff
@@ -32,36 +35,122 @@ const ActorInit Obj_Hsblock_InitVars = {
     (ActorFunc)ObjHsblock_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_8093E33C[] = {
+static f32 focusPos[] = { 85.0f, 85.0f, 0.0f };
```

Comment:
```suggestion
static f32 focusPoss[] = { 85.0f, 85.0f, 0.0f };
```
since it's an array (otherwise looks like a Vec3f, if nothing else).